### ISSUE TITLE
Authenticator is a great product, but addons.mozilla.org is still stuck in 6.3.5, can you update the version of the Firefox extension?

### ISSUE THREAD

**Issue by rocex** [Authenticator-Extension/Authenticator#1204]
### Describe the issue

I found that the https://addons.mozilla.org/zh-CN/firefox/addon/auth-helper/ version is still stuck in 6.3.5, do you have any plans to update it?

### Browser

Firefox

### Browser Version

127

### Extension Version

6.3.5

**Comment by Sneezry** [comment]
We have upgraded to Manifest V3 (MV3) in version 7.0.0 and published it to Chrome to meet the MV2 lifetime deadline before June 2024. This migration involves significant changes, as Chrome and Firefox have different policies for MV3. For instance, Firefox does not support service workers in MV3, while Chrome does not support background scripts. We have spent considerable time ensuring the extension is compatible across different browsers. Since Firefox does not have an urgent timeline for transitioning to MV3, we plan to conduct further testing before releasing version 7.0.0 to Firefox. @mymindstorm will provide more details about the plan when we release the new version to Firefox.
